src/board/mod.rs

```diff
@@ -32,14 +32,14 @@ use ruleset::Ruleset;
 use score::Score;
 use self::point::Point;
```

Comment:
Actually, hold on. What I meant is that I like to keep the setup of two blocks of uses. The first one (alphabetically ordered :stuck_out_tongue_winking_eye:) for stuff from our own repo. And then a second block with the stuff from the stdlib and external crates. So the `rand` stuff needs to go down and `std::sync::Arc` needs to go to the spot where it was (ordered by full name, please).